apiVersion: events.k8s.io/v1
deprecatedCount: 2
deprecatedFirstTimestamp: "2023-03-09T14:28:26Z"
deprecatedLastTimestamp: "2023-03-09T14:29:16Z"
deprecatedSource:
  component: oauth-apiserver-status-controller-statussyncer_authentication
eventTime: null
kind: Event
metadata:
  creationTimestamp: "2023-03-09T14:28:26Z"
  managedFields:
  - apiVersion: v1
    fieldsType: FieldsV1
    fieldsV1:
      f:count: {}
      f:firstTimestamp: {}
      f:involvedObject: {}
      f:lastTimestamp: {}
      f:message: {}
      f:reason: {}
      f:source:
        f:component: {}
      f:type: {}
    manager: authentication-operator
    operation: Update
    time: "2023-03-09T14:29:17Z"
  name: openshift-authentication-operator.174ac689eaf47a19
  namespace: openshift-authentication-operator
  resourceVersion: "9546"
  uid: d84c278e-aecd-455e-a909-5b0929fae905
note: 'Status for clusteroperator/authentication changed: Degraded message changed
  from "APIServerDeploymentDegraded: 1 of 1 requested instances are unavailable for
  apiserver.openshift-oauth-apiserver ()\nIngressStateEndpointsDegraded: No subsets
  found for the endpoints of oauth-server\nOAuthServerServiceEndpointAccessibleControllerDegraded:
  Get \"https://172.30.70.249:443/healthz\": dial tcp 172.30.70.249:443: connect:
  connection refused\nOAuthServerServiceEndpointsEndpointAccessibleControllerDegraded:
  oauth service endpoints are not ready" to "IngressStateEndpointsDegraded: No subsets
  found for the endpoints of oauth-server\nOAuthServerServiceEndpointAccessibleControllerDegraded:
  Get \"https://172.30.70.249:443/healthz\": dial tcp 172.30.70.249:443: connect:
  connection refused\nOAuthServerServiceEndpointsEndpointAccessibleControllerDegraded:
  oauth service endpoints are not ready",Available message changed from "APIServerDeploymentAvailable:
  no apiserver.openshift-oauth-apiserver pods available on any node.\nAPIServicesAvailable:
  apiservices.apiregistration.k8s.io/v1.oauth.openshift.io: not available: failing
  or missing response from https://10.128.0.41:8443/apis/oauth.openshift.io/v1: bad
  status from https://10.128.0.41:8443/apis/oauth.openshift.io/v1: 401\nAPIServicesAvailable:
  apiservices.apiregistration.k8s.io/v1.user.openshift.io: not available: failing
  or missing response from https://10.128.0.41:8443/apis/user.openshift.io/v1: bad
  status from https://10.128.0.41:8443/apis/user.openshift.io/v1: 401\nOAuthServerServiceEndpointAccessibleControllerAvailable:
  Get \"https://172.30.70.249:443/healthz\": dial tcp 172.30.70.249:443: connect:
  connection refused\nOAuthServerServiceEndpointsEndpointAccessibleControllerAvailable:
  endpoints \"oauth-openshift\" not found" to "APIServicesAvailable: apiservices.apiregistration.k8s.io/v1.oauth.openshift.io:
  not available: failing or missing response from https://10.128.0.41:8443/apis/oauth.openshift.io/v1:
  bad status from https://10.128.0.41:8443/apis/oauth.openshift.io/v1: 401\nAPIServicesAvailable:
  apiservices.apiregistration.k8s.io/v1.user.openshift.io: not available: failing
  or missing response from https://10.128.0.41:8443/apis/user.openshift.io/v1: bad
  status from https://10.128.0.41:8443/apis/user.openshift.io/v1: 401\nOAuthServerServiceEndpointAccessibleControllerAvailable:
  Get \"https://172.30.70.249:443/healthz\": dial tcp 172.30.70.249:443: connect:
  connection refused\nOAuthServerServiceEndpointsEndpointAccessibleControllerAvailable:
  endpoints \"oauth-openshift\" not found"'
reason: OperatorStatusChanged
regarding:
  apiVersion: v1
  kind: Namespace
  name: openshift-authentication-operator
  namespace: openshift-authentication-operator
type: Normal
